### PR TITLE
Remove toggle from CLI documentation

### PR DESCRIPTION
**Pull request by bszwarc**
**Description**

Changes proposed in this pull request:

- Remove the unnecessary toggle from the `install` command
- Improve wording and formatting of command description

**Related issue(s)**
See also #251

### CODE REVIEW

File: docs/gen-docs/kyma_version.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ kyma version [flags]
...
   -c, --client   Client version only (no server required)
-  -h,  --help     Displays help for the command.
+  -h, --help     Displays help for the command.
 ```
 
 ### Options inherited from parent commands
 
 ```
-      --kubeconfig string   Specifies the path to the kubeconfig file. (default KUBECONFIG environment variable or "/$HOME/.kube/config" if KUBECONFIG is not set)
+      --kubeconfig string   Specifies the path to the kubeconfig file. Use the default KUBECONFIG environment variable or /$HOME/.kube/config if KUBECONFIG is not set.
```
Comment (clebs) [inline docs/gen-docs/kyma_version.md]: The change to this sentence is a bit misleading, I understand it as: I (the user) have to use the KUBECONFIG or the /HOME... Also "default KUBECONFIG" is confusing, there is no such thing.
It should somehow make clear that the CLI will automatically picks either the KUBECONFIG or the /$HOME/.kube/config if not KUBECONFIG was found when this flag is not explicitly set (so it is the default value).

File: cmd/kyma/test/run/cmd.go [hunk 2]
```diff
@@ -32,18 +32,17 @@ func NewCmd(o *options) *cobra.Command {
...
-To execute all test defintions, run the"example-test" test suite:
-kyma test run -n example-test
 `,
 		RunE:    func(_ *cobra.Command, args []string) error { return cmd.Run(args) },
 		Aliases: []string{"r"},
 	}
 
-	cobraCmd.Flags().StringVarP(&o.Name, "name", "n", "", "Name of the new test suite")
-	cobraCmd.Flags().Int64VarP(&o.ExecutionCount, "count", "c", 1, "Number of execution rounds for each test in the suite. You cannot configure this value in parallel with max-retries")
+	cobraCmd.Flags().StringVarP(&o.Name, "name", "n", "", "Name of the new test suite. If you don't specify the value for the `-n` flag, the name of the test suite name will be autogenerated.")
```
Comment (kazydek) [inline cmd/kyma/test/run/cmd.go]:
```suggestion
	cobraCmd.Flags().StringVarP(&o.Name, "name", "n", "", "Name of the new test suite. If you don't specify the value for the `-n` flag, the name of the test suite will be autogenerated.")
```

File: cmd/kyma/test/status/cmd.go [hunk 3]
```diff
@@ -30,18 +30,16 @@ func NewCmd(o *options) *cobra.Command {
 
 	cobraCmd := &cobra.Command{
 		Use:   "status <test-suite-1> <test-suite-2> ... <test-suite-N>",
-		Short: "Shows the status of a test suite and it's related test executions",
+		Short: "Shows the status of a test suite and its related test executions",
```
Comment (kazydek) [inline cmd/kyma/test/status/cmd.go]:
```suggestion
		Short: "Shows the status of a test suite and related test executions",
```

File: docs/gen-docs/kyma_test.md [hunk 4]
```diff
@@ -27,5 +27,5 @@ Use this command to run tests on a provisioned Kyma cluster.
 * [kyma test delete](kyma_test_delete.md)	 - Deletes test suites available for a provisioned Kyma cluster.
 * [kyma test list](kyma_test_list.md)	 - Lists test suites available for a provisioned Kyma cluster.
 * [kyma test run](kyma_test_run.md)	 - Runs tests on a Kyma cluster.
-* [kyma test status](kyma_test_status.md)	 - Shows the status of a test suite and it's related test executions
+* [kyma test status](kyma_test_status.md)	 - Shows the status of a test suite and its related test executions
```
Comment (kazydek) [inline docs/gen-docs/kyma_test.md]:
```suggestion
* [kyma test status](kyma_test_status.md)	 - Shows the status of a test suite and related test executions.
```

File: docs/gen-docs/kyma_test_status.md [hunk 5]
```diff
@@ -1,20 +1,17 @@
 ## kyma test status
 
-Shows the status of a test suite and it's related test executions
+Shows the status of a test suite and its related test executions
```
Comment (kazydek) [inline docs/gen-docs/kyma_test_status.md]:
```suggestion
Shows the status of a test suite and related test executions.
```

File: docs/gen-docs/kyma_test_status.md [hunk 6]
```diff
@@ -1,20 +1,17 @@
...
 
 Use this command to display the status of a test suite and related test executions.
 
 If you don't provide any arguments, the status of all test suites will be printed.
 
-To print the status of all test suites, run:
+To print the status of all test suites, `run kyma test status`
 
-    kyma test status
+To print the status of specific test cases, `run kyma test status testSuiteOne testSuiteTwo`
```
Comment (kazydek) [inline docs/gen-docs/kyma_test_status.md]:
```suggestion
To print the status of specific test cases, `run kyma test status testSuiteOne testSuiteTwo`.
```

File: docs/gen-docs/kyma_provision_minikube.md [hunk 7]
```diff
@@ -18,13 +18,14 @@ kyma provision minikube [flags]
   -h, --help                         Displays help for the command.
       --hypervVirtualSwitch string   Specifies the Hyper-V switch version if you choose Hyper-V as the driver.
       --memory string                Specifies RAM reserved for installation. (default "8192")
+      --profile string               Specifies the minikube profile. (default "minikube")
```
Comment (kazydek) [inline docs/gen-docs/kyma_provision_minikube.md]:
```suggestion
      --profile string               Specifies the Minikube profile. (default "minikube")
```

File: cmd/kyma/test/status/cmd.go [hunk 8]
```diff
@@ -30,18 +30,16 @@ func NewCmd(o *options) *cobra.Command {
...
 		Long: `Use this command to display the status of a test suite and related test executions.
 
 If you don't provide any arguments, the status of all test suites will be printed.
 
-To print the status of all test suites, run:
+To print the status of all test suites, `+"`run kyma test status`"+`
```
Comment (kazydek) [inline cmd/kyma/test/status/cmd.go]:
```suggestion
To print the status of all test suites, `+"`run kyma test status`"+`.
```

File: cmd/kyma/test/run/cmd.go [hunk 9]
```diff
@@ -32,18 +32,17 @@ func NewCmd(o *options) *cobra.Command {
...
+	cobraCmd.Flags().Int64VarP(&o.ExecutionCount, "count", "c", 1, "Number of execution rounds for each test in the suite. It is not possible configure this value in parallel with `max-retries`.")
```
Comment (kazydek) [inline cmd/kyma/test/run/cmd.go]:
```suggestion
	cobraCmd.Flags().Int64VarP(&o.ExecutionCount, "count", "c", 1, "Number of execution rounds for each test in the suite. It is not possible to configure this value in parallel with `max-retries`.")
```

File: docs/gen-docs/kyma_provision_minikube.md (hunk 7 above)
Comment (kazydek) [inline docs/gen-docs/kyma_provision_minikube.md]:
```suggestion
      --profile string               Specifies the Minikube profile. (default "minikube")
```

File: cmd/kyma/test/status/cmd.go [hunk 10]
```diff
@@ -30,18 +30,16 @@ func NewCmd(o *options) *cobra.Command {
 
 	cobraCmd := &cobra.Command{
 		Use:   "status <test-suite-1> <test-suite-2> ... <test-suite-N>",
-		Short: "Shows the status of a test suite and it's related test executions",
+		Short: "Shows the status of a test suite and related test executions",
```
Comment (kazydek) [inline cmd/kyma/test/status/cmd.go]:
```suggestion
		Short: "Shows the status of a test suite and related test executions.",
```

File: docs/gen-docs/kyma_test.md (hunk 4 above)
Comment (kazydek) [inline docs/gen-docs/kyma_test.md]:
```suggestion
* [kyma test status](kyma_test_status.md)	 - Shows the status of a test suite and related test executions.
```

File: docs/gen-docs/kyma_test_status.md (hunk 5 above)
Comment (kazydek) [inline docs/gen-docs/kyma_test_status.md]:
```suggestion
Shows the status of a test suite and related test executions.
```

File: docs/gen-docs/kyma_test_status.md [hunk 11]
```diff
@@ -1,20 +1,17 @@
...
+Shows the status of a test suite and its related test executions
 
 ### Synopsis
 
 Use this command to display the status of a test suite and related test executions.
 
 If you don't provide any arguments, the status of all test suites will be printed.
 
-To print the status of all test suites, run:
+To print the status of all test suites, `run kyma test status`
```
Comment (kazydek) [inline docs/gen-docs/kyma_test_status.md]:
```suggestion
To print the status of all test suites, `run kyma test status`.
```

File: docs/gen-docs/kyma_test_status.md (hunk 6 above)
Comment (kazydek) [inline docs/gen-docs/kyma_test_status.md]:
```suggestion
To print the status of specific test cases, `run kyma test status testSuiteOne testSuiteTwo`.
```

File: docs/gen-docs/kyma_test_status.md (hunk 11 above)
Comment (bszwarc) [reply docs/gen-docs/kyma_test_status.md]: 👍

File: docs/gen-docs/kyma_test_status.md (hunk 6 above)
Comment (bszwarc) [reply docs/gen-docs/kyma_test_status.md]: 👍

File: docs/gen-docs/kyma_test.md (hunk 4 above)
Comment (bszwarc) [reply docs/gen-docs/kyma_test.md]: Done :)

File: docs/gen-docs/kyma_test_status.md [hunk 12]
```diff
@@ -1,20 +1,17 @@
...
+Shows the status of a test suite and its related test executions.
 
 ### Synopsis
 
 Use this command to display the status of a test suite and related test executions.
 
 If you don't provide any arguments, the status of all test suites will be printed.
 
-To print the status of all test suites, run:
+To print the status of all test suites, `run kyma test status`
```
Comment (kazydek) [inline docs/gen-docs/kyma_test_status.md]:
```suggestion
To print the status of all test suites, `run kyma test status`.
```

File: docs/gen-docs/kyma_test_status.md [hunk 13]
```diff
@@ -1,20 +1,17 @@
 ## kyma test status
 
-Shows the status of a test suite and it's related test executions
+Shows the status of a test suite and its related test executions.
```
Comment (kazydek) [inline docs/gen-docs/kyma_test_status.md]:
```suggestion
Shows the status of a test suite and related test executions.
```

File: docs/gen-docs/kyma_test.md (hunk 4 above)
Comment (kazydek) [inline docs/gen-docs/kyma_test.md]:
```suggestion
* [kyma test status](kyma_test_status.md)	 - Shows the status of a test suite and related test executions.
```

File: docs/gen-docs/kyma_test_status.md [hunk 14]
```diff
@@ -1,20 +1,17 @@
...
 
 Use this command to display the status of a test suite and related test executions.
 
 If you don't provide any arguments, the status of all test suites will be printed.
 
-To print the status of all test suites, run:
+To print the status of all test suites, `run kyma test status`
 
-    kyma test status
+To print the status of specific test cases, `run kyma test status testSuiteOne testSuiteTwo`
```
Comment (kazydek) [inline docs/gen-docs/kyma_test_status.md]:
```suggestion
To print the status of specific test cases, `run kyma test status testSuiteOne testSuiteTwo`.
```

File: docs/gen-docs/kyma_test.md [hunk 15]
```diff
@@ -27,5 +27,5 @@ Use this command to run tests on a provisioned Kyma cluster.
 * [kyma test delete](kyma_test_delete.md)	 - Deletes test suites available for a provisioned Kyma cluster.
 * [kyma test list](kyma_test_list.md)	 - Lists test suites available for a provisioned Kyma cluster.
 * [kyma test run](kyma_test_run.md)	 - Runs tests on a Kyma cluster.
-* [kyma test status](kyma_test_status.md)	 - Shows the status of a test suite and it's related test executions
+* [kyma test status](kyma_test_status.md)	 - Shows the status of a test suite and related test executions
```
Comment (kazydek) [inline docs/gen-docs/kyma_test.md]:
```suggestion
* [kyma test status](kyma_test_status.md)	 - Shows the status of a test suite and related test executions.
```

**Pull request merged**